src/Tasks/Microsoft.NET.Build.Tasks/PreprocessPackageDependenciesDesignTime.cs

```diff
@@ -440,6 +522,26 @@ private class AssemblyMetadata : PackageMetadata
             }
         }
 
+        /// <summary>
+        /// Represents metadata for a Reference item that we want to pretend was resolved as a
+        /// standard NuGet package assembly.
+        /// </summary>
+        private class PreResolvedNetStandardAssemblyMetadata : PackageMetadata
+        {
+            public PreResolvedNetStandardAssemblyMetadata(string name, string path)
+                : base(
+                      type: DependencyType.Assembly,
+                      dependencies: null,
+                      isTopLevelDependency: false,
+                      name: name,
+                      version: string.Empty,
+                      path: path,
+                      resolved: true,
+                      isImplicitlyDefined: false)
```

Comment:
@abpiskunov 

IIRC, this controls whether the item can be deleted via IDE. However, there was some discussion which I think was never resolved around the fact that essentially anything in a target cannot be deleted in IDE and relying on metadata for it is problematic.